inc/classes/preload/class-homepage.php

```diff
@@ -18,6 +18,33 @@ class Homepage extends Abstract_Preload {
 	 * @return void
 	 */
 	public function preload( $urls ) {
+		foreach ( $urls as $home_url ) {
+			$urls      = $this->get_urls( $home_url );
```

Comment:
This `$urls` is confusing with the one above. It should be something like `$home_urls` imho.